dm/ctl/common/config.go

```diff
@@ -117,7 +125,19 @@ func (c *Config) configFromFile(path string) error {
 }
 
 // adjust adjusts configs
-func (c *Config) adjust() {
+func (c *Config) adjust() error {
+	if c.RPCTimeoutStr == "" {
+		c.RPCTimeoutStr = defaultRPCTimeout
+	}
+	timeout, err := time.ParseDuration(c.RPCTimeoutStr)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if timeout <= time.Duration(0) {
```

Comment:
Is a too small timeout value allowed? like _1ms_.

Reply:
added some comment in sample `dmctl.toml`, user should have known the meaning of `rpc-time` if they configure it, so I think that checking the timeout is not negative is enough.